src/main/java/com/example/java/test/junior/developer/model/Language.java

```diff
@@ -26,5 +25,11 @@ public class Language {
     @Column(name = "language", nullable = false)
     public String name;
 
+    @OneToMany(cascade = CascadeType.ALL,
+            mappedBy = "language",
+            fetch = FetchType.EAGER)
+    private List<Category> list_categories;
```

Comment:
Java Code Convention violation - NEVER use underscore in names in Java, only camel case.